.pydocstyle

```diff
@@ -1,2 +1,2 @@
 [pydocstyle]
-ignore = D100,D101,D102,D103,D104,D105,D200,D203,D204,D205,D207,D208,D209,D210,D212,D213,D214,D301,D302,D400,D401,D402,D403,D404,D405,D406,D407,D412,D413
+ignore = D100,D101,D102,D103,D104,D105,D200,D203,D204,D205,D207,D208,D209,D210,D212,D213,D214,D400,D401,D402,D403,D404,D405,D406,D407,D412,D413
```

Comment:
this one was likely due to the fact that i had to edit it manually to resolve a conflict.